File: Android/Sample/app.ok-if-stub/src/main/java/net/wequick/example/small/appok_if_stub/NotifyResultActivity.java
package net.wequick.example.small.appok_if_stub;

import android.os.Bundle;
import android.support.v4.app.NotificationManagerCompat;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class NotifyResultActivity extends AppCompatActivity {

    private int mNotificationId;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pending);

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        mNotificationId = getIntent().getIntExtra("notification_id", 0);
        TextView textView = (TextView) findViewById(R.id.notification_id_label);
        textView.setText(mNotificationId + "");

        Button removeButton = (Button) findViewById(R.id.remove_notification_button);
        removeButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                NotificationManagerCompat.from(NotifyResultActivity.this).cancel(mNotificationId);
            }
        });
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }

    @Override
    public void finish() {
        NotificationManagerCompat.from(this).cancel(mNotificationId);
        super.finish();
    }
}

File: Android/Sample/app.ok-if-stub/src/main/java/net/wequick/example/small/appok_if_stub/NotifyResultActivity.kt
package net.wequick.example.small.appok_if_stub

import android.os.Bundle
import android.support.v4.app.NotificationManagerCompat
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem
import android.view.View
import android.widget.Button
import android.widget.TextView

class NotifyResultActivity : AppCompatActivity() {

    private var mNotificationId: Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pending)

        supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        mNotificationId = intent.getIntExtra("notification_id", 0)
        val textView = findViewById(R.id.notification_id_label) as TextView
        textView.text = mNotificationId.toString() + ""

        val removeButton = findViewById(R.id.remove_notification_button) as Button
        removeButton.setOnClickListener { NotificationManagerCompat.from(this@NotifyResultActivity).cancel(mNotificationId) }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }

    override fun finish() {
        NotificationManagerCompat.from(this).cancel(mNotificationId)
        super.finish()
    }
}
